src/UnityPackage/Assets/Web3Unity/Scripts/Prefabs/AllNftsInfura/AllNfts.cs

```diff
@@ -0,0 +1,94 @@
+using Newtonsoft.Json;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Networking;
+
+public class AllNfts : MonoBehaviour
+{
+    /*
+    Infura has come out swinging with a potential replacement for blockchain indexing
+    Link: https://docs.infura.io/infura/infura-expansion-apis/nft-api
+    The included script will include everything you need to start shaping your own custom API calls to their endpoints and query data like:
+    Transfers
+    Ownership
+    Metadata
+    Collections
+    Market Data
+    In the included example, we query all the owners for a specific token on the Cronos testnet.
+    */
+    // Feel free to change all of this. It's purely educational and nothing more than a proof of concept
+    private string APICall;
+    // Add your Infura API Key and Secret API KEY:API SECRET
+    private string APIKey;
+    // Which Chain is your contract deployed on?
+    public string ChainID;
+    // What is your NFT Contract Address?
+    public string TokenContract;
+    // What tokenID do you want to check against?
+    public string TokenID;
+
+
+    [Serializable]
+    public class TokenData
```

Comment:
Nested-type declarations go before field declarations if I'm not mistaken.

Reply:
closed bud, no need :)

Reply:
ok, no problem